.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: php
+php:
+  - 7.1
+  - 7.3
+before_script:
+  - composer self-update
```

Comment:
not needed

Reply:
https://docs.travis-ci.com/user/languages/php/#installing-composer-packages
Correct, line was deleted.